tests/helper/tests/key_val_in_file.py

```diff
@@ -15,7 +22,10 @@ def _get_content_remote_file(client, fname):
 
 def _get_key_val_from_content(content, dict, fname):
     """ Validates if key/val are present in content """
+    present = False
     for k in dict:
         for line in content.split('\n'):
             if line.startswith(k):
```

Comment:
Just checking if a line starts with a given key may not be robust, especially thinking about the use case you are using this function with. 

For example, there are unit directives like `ExecStart`, `ExecStartPre` or `ExecStartPost`, where is if condition would also work with. This creates false positives.

Reply:
Please find the last commit...

Reply:
for the `str.split` to get the first word based on `index 0` and to cut everything after a pattern of `=` or `:` (to have it more generic). But I see your problem, now. I'll rewrite this to only have a single regex pattern.

Reply:
Ok, adjusted.